### ISSUE TITLE
Ship with Google Scholar friendly config in islandora_defaults

### ISSUE THREAD

**Issue by dannylamb** [Islandora/documentation#1198]
We can ship with default configuration to aid in Google Scholar indexing, which would amount to two things:

1. Ship with the `sitemap` module, or provide a view that lists articles by year (with NO PAGER, GIVE US ALL THE CONTENTS)
1. Ship with `metatag` configuration to provide the basics using Highwire Press tags.

**Comment by bryjbrown** [comment]
Here's a link to the inclusion documentation that has lots of info on appropriate metatags: https://scholar.google.com/intl/en/scholar/inclusion.html#indexing

**Comment by bryjbrown** [comment]
One of the ideas we've had in the IR IG for finding a use for the name "Scholar" without necessarily recreating all the functionality and structure of 7.x Scholar is to make 8.x Scholar more of a meta module that uses the [Checklist API](https://www.drupal.org/project/checklistapi) and [Tour](https://www.drupal.org/docs/8/core/modules/tour) modules to ensure that someone who wants to use Islandora 8.x as an IR platform has all the best practices configured. Making sure metatags, sitemap and any Google Scholar requirements are set up could be something this module checks for.

**Comment by rosiel** [comment]
From what I can see, metatag (contrib) and rdf (core) both do very similar things, i.e. configuring mappings between Drupal fields and RDF predicates.

Do we know how well they'd work "together" (in the sense of: how similar are the things that they're doing, do we need both, and can we "synchronize" mappings between them? Or does it make sense to maintain duplicate mappings because they have different purposes? How can we communicate/expose this to users?)

As you mentioned, metatag is immediately useful because we have harvesters. As mentioned in the OAI ticket #1192 we also have harvesters waiting to get DCE, which metatag provides (and has a UI for :) ). I'm not sure yet who will be reading the "fancy rdf" that's going into Fedora.

Tagging in @alxp who knows lots about metatag.

**Comment by rosiel** [comment]
After talking with Danny, it sounds like yes, we need Metatag. 

It would be good to open the door to more (to overload a word) _contextual_ mappings, whether that's with templates or different endpoints or whatnot. But that should probably not get in the way of shipping with metatag, which is out-of-the-box and makes the Alphabet Overlords happy.

**Comment by dannylamb** [comment]
Heads up: https://www.drupal.org/project/metatag_google_scholar

Looks like it could work for us.

**Comment by rosiel** [comment]
It'd be great to reuse something, even though it's beta1 and hasn't been updated in 2 years. Assuming it does what it claims to, does that jive with what the IRIG knows from interacting with actual humans at Google?

**Comment by sarah-lippincott** [comment]
Yes, this module's functionality would be appropriate. Google Scholar expects repositories to expose meta tags in High Wire Press format. (The full guidelines are at https://scholar.google.com/intl/en/scholar/inclusion.html#indexing, and confirmed by Monica Westin of Google Scholar at a recent IRIG meeting.) The module appears to be missing at least one of the meta tags recognized by Google (e.g., citation_conference_title), but otherwise matches the functionality requirements.

**Comment by seth-shaw-unlv** [comment]
Note, the [metatag_google_scholar module](https://www.drupal.org/project/metatag_google_scholar) just got a release version on June 1st.

**Comment by rosiel** [comment]
I'm adding the documentation tag to encourage us to keep a documented knowledge base of what we know about interacting wtih google scholar. Even if it's in islandora defaults, it'll need to be managed by knowledgeable site maintainers.
